code/modules/cargo/exports/parts.dm

```diff
@@ -19,3 +19,8 @@
 	cost = 2000
 	unit_name = "deactivated alien deconstruction drone"
 	export_types = list(/obj/item/deactivated_swarmer)
+
+/datum/export/pyroclastic_anomaly
+	cost = 10000
+	unit_name = "A rare disarmed anomaly core"
+	export_types = list(/obj/effect/anomaly/pyro)
```

Comment:
Is this some kind of experiment to see if anyone bothers to read the diff?

Reply:
nice